otherlibs/unix/unix.mli

```diff
@@ -543,6 +543,28 @@ val map_file :
   validation fails.
   @since 4.06.0 *)
 
+val flush_mapped_file :
```

Comment:
Does `UnixLabels` also need a corresponding function added to it?

Reply:
I'm afraid so.  Added `UnixLabels.flush_mapped_file`.